packages/client/package.json

```diff
@@ -6,8 +6,10 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.1.1",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.2.2",
```

Comment:
We don't single-handedly install dependencies without alignment with mentors and the team.
You should build the component yourself, @Hailegebriel22  😅